lib/eu_central_bank.rb

```diff
@@ -67,6 +69,7 @@ def exchange_with(from, to_currency, date=nil)
   end
 
   def get_rate(from, to, date = nil)
+    check_currency_available(from) and check_currency_available(to)
```

Comment:
Yeah, at the beginning the detection was there and then I liked to have a one-liner.